accepted/2021/experimental-package-shading.md

```diff
@@ -0,0 +1,73 @@
+# Experiment: NuGet Package Shading
+
+(intro and scenarios to be added)
+
```

Comment:
We need probably a "Problem" statement section.  I am assuming that will be added as part of the Intro?

Reply:
Yes, it will.